client/js/options.js

```diff
@@ -86,9 +88,15 @@ settings.on("change", "input, select, textarea", function() {
 			// otherwise, users get notifications for everything
 			return h !== "";
 		});
-	} else if (name === "showSeconds") {
+	} else if (name === "showSeconds" || name === "use12hClock") {
 		chat.find(".msg > .time").each(function() {
-			$(this).text(tz($(this).parent().data("time")));
+			const time = $(this).parent().data("time");
+
+			$(this).text(tz(time));
+
+			if (name === "use12hClock") {
+				$(this).attr("aria-label", localetime(time));
```

Comment:
`localetime` requires `options` and `options` requires `localetime` 😰 Not sure we want to go there :/